src/main/java/one/util/streamex/MoreCollectors.java

```diff
@@ -168,6 +169,188 @@ private MoreCollectors() {
         }, Function.identity(), set -> set.size() == size, UNORDERED_ID_CHARACTERISTICS);
     }
 
+    /**
+     * Returns a {@code Collector} that accumulates elements into a {@code Map}
+     * whose keys and values are taken from {@code Map.Entry}.
```

Comment:
I suggest adding a statement like. `There are no guarantees on the type or serializability of the Map returned; if more control over the returned Map is required, use entriesToCustomMap(Supplier).` And also `Returned {@code Map} is guaranteed to be modifiable`. See one.util.streamex.EntryStream#toMap(). Similar for `entriesToMap(combiner)`.